docs/teams/mentor-guidance.md

```diff
@@ -0,0 +1,224 @@
+# Mentor Guidance
+
+A rough guide on how to encourage learners.
+
+## Finding a level
+
+It’s important to pitch your support at the right level for the person
+you’re helping, so that they’re able to absorb what you’re saying. One
+way to think about this is adding branches to a trunk of knowledge; if
+the trunk is a sapling then adding a large bough with many branches will
+crush the sapling. Conversely, presenting a bud of information can be
+frustrating for both parties if the person you’re helping is more
+experienced.
+
+For complete novices, or when very first introducing a new topic (even
+to an experienced audience), this means that you should try to present
+an idea in its very simplest form. Try to think about the essence of
+what makes the idea important, rather than the details of how it works.
+A good example of this is the idea of “[*lies to children*][lies-to-children]”.
```

Comment:
Might be worth checking if `“` and `”` render correctly in mkdocs.

Alternatively: `"`

Reply:
They render fine :+1:. I'd kinda hoped that mkdocs would even convert plain ones, but it doesn't :disappointed:.